<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SoundLibraryImportDialog_UI</class>
 <widget class="QDialog" name="SoundLibraryImportDialog_UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>10</y>
     <width>511</width>
     <height>391</height>
    </rect>
   </property>
   <widget class="QWidget" name="Internet" >
    <attribute name="title" >
     <string>Internet</string>
    </attribute>
    <widget class="QWidget" name="widget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="UpdateListBtn" >
        <property name="text" >
         <string>Update list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="m_pDrumkitTree" >
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="indentation" >
         <number>15</number>
        </property>
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="itemsExpandable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget" >
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>10</y>
       <width>191</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="SoundLibraryNameLbl" >
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text" >
         <string>Sound Library Name</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SoundLibraryInfoLbl" >
        <property name="text" >
         <string>Sound library Info</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="margin" >
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="AuthorLbl" >
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text" >
         <string>Author...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DownloadBtn" >
        <property name="text" >
         <string>Download and install</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>Local file</string>
    </attribute>
    <widget class="QWidget" name="widget" >
     <property name="geometry" >
      <rect>
       <x>11</x>
       <y>21</y>
       <width>491</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QPushButton" name="BrowseBtn" >
        <property name="text" >
         <string>Browse...</string>
        </property>
        <property name="flat" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="InstallBtn" >
        <property name="text" >
         <string>Install</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLineEdit" name="SoundLibraryPathTxt" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
